pysrc/models/metanome_run.py

```diff
@@ -270,6 +275,9 @@ def run_metanome(configuration: MetanomeRunConfiguration, output_fname: str, pip
     output_rule = f'file:{output_fname}'
     allowed_gb: int = 6
 
+    # Calculate File Statistics
+    source_files_column_statistics = [file_column_statistics(f, is_baseline=configuration.is_baseline) for f in configuration.source_files]
+    source_files_column_statistics = list(chain(*source_files_column_statistics)) # Concatenate subLists into a single mainList
```

Comment:
```suggestion
    source_files_column_statistics = [stats for f in configuration.source_files for stats in file_column_statistics(f, is_baseline=configuration.is_baseline)]
```

Not sure about that one. I like it better this way but you can also keep it the old way. When you're changing it, remember to remove the then unused import.